backward.hpp

```diff
@@ -3810,11 +3816,15 @@ class SourceFile {
     return paths;
   }
 
-  const std::vector<std::string> &get_paths_from_env_variable() {
+  static std::vector<std::string> &get_mutable_paths_from_env_variable() {
     static std::vector<std::string> paths = get_paths_from_env_variable_impl();
```

Comment:
So this makes this variable mutable. I know it's C++ and everything is fair game. But this does mean you can mutate this vector from a different thread.

Reply:
Using this function, yes; though this function is private so only via the public mutator function. We can add a mutex in that function easily enough if desired, though this is not a function I see being called outside of `setup()` before threading takes place. If you want a mutex just let me know and I'll use a scoped lock in the accessor function.

Reply:
In this case, that mutator function is the added `add_paths_to_env_variable_impl` function